mautic/mautic
# Add language support for forms

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

PR added support language for forms. 
You can use translations for validate default message and please wait message

![image](https://github.com/mautic/mautic/assets/462477/31c77b9d-51dd-4789-af9e-2f67c1c461f7)



1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create a form with the required fields and setup FR language
3.  Try submit
4.  See FR default validation string
5. Same for please wait text on submit form
